ControlWindowWPF/ControlWindowWPF/Resources/Chinese.xaml

```diff
@@ -503,27 +503,27 @@
     <system:String x:Key="VMCProtocolReceiverSettingWindow_Eye">眼</system:String>
     <system:String x:Key="VMCProtocolReceiverSettingWindow_RightFinger">右手指</system:String>
     <system:String x:Key="VMCProtocolReceiverSettingWindow_LeftFinger">左手指</system:String>
-    <system:String x:Key="VMCProtocolReceiverSettingWindow_UnselectBones">Unselect bones</system:String>
-    <system:String x:Key="VMCProtocolReceiverSettingWindow_Port">Port :</system:String>
-    <system:String x:Key="VMCProtocolReceiverSettingWindow_Delay">Delay :</system:String>
-    <system:String x:Key="VMCProtocolReceiverSettingWindow_FixHandBone">Fix hand bone</system:String>
-    <system:String x:Key="VMCProtocolReceiverSettingWindow_UseBonePosition">Use bone position</system:String>
+    <system:String x:Key="VMCProtocolReceiverSettingWindow_UnselectBones">取消选定骨关节</system:String>
+    <system:String x:Key="VMCProtocolReceiverSettingWindow_Port">端口：</system:String>
+    <system:String x:Key="VMCProtocolReceiverSettingWindow_Delay">延迟：</system:String>
+    <system:String x:Key="VMCProtocolReceiverSettingWindow_FixHandBone">固定手骨</system:String>
```

Comment:
This "Fix" means modify(修整？).
So I think correct translate is 修整手骨 or something.
Can you correct this?

Reply:
@sh-akira I translated this string based on English. So, the word "fix" here has different meanings in different contexts. According to your Japanese translation (手のボーンを修正), it could be better translated as "correct" in English and "修正" in Chinese?

Reply:
Yes, I think you are right.
I'll fix English translate.

Reply:
@sh-akira I've fixed the Chinese translation.